specification/components/schemas/codeable/Language.yaml

```diff
@@ -0,0 +1,32 @@
+type: object
+description: Human language.
+required:
+  - coding
+properties:
+  coding:
+    type: array
+    description: Exactly one language code.
+    minItems: 1
+    maxItems: 1
```

Comment:
opposite to previous comment, is this counting from 0? if so should this be 0?

Reply:
PDS API